JuiceMaker/JuiceMaker/Controller/MakeJuiceViewController.swift

```diff
@@ -0,0 +1,171 @@
+
+import UIKit
+
+enum JuiceName: CustomStringConvertible{
+    case strawberryBanana
+    case mangoKiwi
+    case strawberry
+    case banana
+    case pineapple
+    case kiwi
+    case mango
+    
+    var description: String {
+        switch self {
+        case .strawberryBanana:
+            return "딸바"
+        case .mangoKiwi:
+            return "망키"
+        case .strawberry:
+            return "딸기"
+        case .banana:
+            return "바나나"
+        case .pineapple:
+            return "파인애플"
+        case .kiwi:
+            return "키위"
+        case .mango:
+            return "망고"
+        }
+    }
+}
+
+class MakeJuiceViewController: UIViewController, sendBackDelegate {
+
+    @IBOutlet weak var strawberryLabel: UILabel!
+    @IBOutlet weak var bananaLabel: UILabel!
+    @IBOutlet weak var pineappleLabel: UILabel!
+    @IBOutlet weak var kiwiLabel: UILabel!
+    @IBOutlet weak var mangoLabel: UILabel!
+    
+    let myJuiceMaker = JuiceMaker()
+
+    @IBAction func orderStrawberryBananaJuice(_ sender: UIButton) {
```

Comment:
IBAction의 경우 target에 발생한 이벤트를 나타내주는 네이밍이면 좋아요
```suggestion
    @IBAction func StrawberryBananaJuiceButtonDidTap(_ sender: UIButton) {
```